python/gui/symbology-ng/qgsrendererpropertiesdialog.sip

```diff
@@ -1,93 +1,141 @@
-     * to re-synchronize with the variables.
-     */
+
     void layerVariablesChanged();
+%Docstring
+ Emitted when expression context variables on the associated
+ vector layers have been changed. Will request the parent dialog
+ to re-synchronize with the variables.
+%End
 
-    /**
-     * Emitted when something on the widget has changed.
-     * All widgets will fire this event to notify of an internal change.
-     */
     void widgetChanged();
+%Docstring
+ Emitted when something on the widget has changed.
+ All widgets will fire this event to notify of an internal change.
+%End
 
-    /**
-     * Emit when you require a panel to be show in the interface.
-     * @param panel The panel widget to show.
-     * @note If you are connected to this signal you should also connect
-     * given panels showPanel signal as they can be nested.
-     */
-    void showPanel( QgsPanelWidget* panel );
+    void showPanel( QgsPanelWidget *panel );
+%Docstring
+ Emit when you require a panel to be show in the interface.
+ \param panel The panel widget to show.
+.. note::
+
+   If you are connected to this signal you should also connect
+ given panels showPanel signal as they can be nested.
+%End
 
   public slots:
-    //! called when user changes renderer type
     void rendererChanged();
+%Docstring
+called when user changes renderer type
+%End
 
-    //! Apply the changes from the dialog to the layer.
     void apply();
+%Docstring
+Apply the changes from the dialog to the layer.
+%End
 
-    //! Apply and accept the changes for the dialog.
     void onOK();
+%Docstring
+Apply and accept the changes for the dialog.
+%End
+
+    void openPanel( QgsPanelWidget *panel );
+%Docstring
+ Open a panel or dialog depending on dock mode setting
+ If dock mode is true this method will emit the showPanel signal
+ for connected slots to handle the open event.
+
+ If dock mode is false this method will open a dialog
+ and block the user.
+
+ \param panel The panel widget to open.
+%End
 
-    /**
-     * Open a panel or dialog depending on dock mode setting
-     * If dock mode is true this method will emit the showPanel signal
-     * for connected slots to handle the open event.
-     *
-     * If dock mode is false this method will open a dialog
-     * and block the user.
-     *
-     * @param panel The panel widget to open.
-     */
-    void openPanel( QgsPanelWidget* panel );
 
   protected:
-    /**
-     * Connect the given slot to the value changed event for the set of widgets
-     * Each widget is checked for type and the common type of signal is connected
-     * to the slot.
-     *
-     * @param widgets The list of widgets to check.
-     * @param slot The slot to connect to the signals.
-     */
-    void connectValueChanged( const QList<QWidget *>& widgets, const char *slot );
 
+    void connectValueChanged( const QList<QWidget *> &widgets, void ( QgsRendererPropertiesDialog::*slot )() );
```

Comment:
some hints maybe [here](https://github.com/baoboa/pyqt5/blob/master/sip/QtQml/qqmlproperty.sip#L79-L92) and in [extension API](http://pyqt.sourceforge.net/Docs/PyQt5/extension_api.html)